config/application.php

```diff
@@ -1,7 +1,6 @@
 <?php
 /**
- * Your base production configuration goes in this file. Environment-specific
- * overrides go in their respective config/environments/{{WP_ENV}}.php file.
+ * Your base production configuration goes in this file.
```

Comment:
Do we want to retain support for environment-level config files? Or would all of that stuff be identified by the PANTHEON_ENVIRONMENT anyway, and therefore be handled in other ways than what Bedrock does?

Reply:
Added a note about those files here: https://github.com/pantheon-systems/wordpress-composer-managed/pull/9#issuecomment-1192004544